### ISSUE TITLE
[Help] image file-upload already error?

### ISSUE THREAD

**Issue by admspro** [Requarks/wiki#563]
Version : Master (Or Last Release Version)

Action (Step)

1. Image Upload (example-1.png)
2. Image Upload (example-2.png) (or Other File,)

#### --> Error ( Upload Error: File example-1.png already exists)

Help Please...

1. Image Upload (example-1.png) --> Success

2. Other Image Upload (example-2.png) --> Error

![image](https://user-images.githubusercontent.com/32889153/42251068-0d43a3aa-7f70-11e8-904c-1bdcd21e3911.png)

![image](https://user-images.githubusercontent.com/32889153/42251101-369a4902-7f70-11e8-8f98-61fa9c6996eb.png)

my config.yml file
```
title: Wiki
host: 'http://192.168.0.65'
port: 80
paths:
  repo: ./repository
  data: ./datafolder
uploads:
  maxImageFileSize: 300
  maxOtherFileSize: 100
lang: en
langRtl: false
public: false
auth:
  defaultReadAccess: false
  local:
    enabled: true
  google:
    enabled: false
    clientId: GOOGLE_CLIENT_ID
    clientSecret: GOOGLE_CLIENT_SECRET
  microsoft:
    enabled: false
    clientId: MS_APP_ID
    clientSecret: MS_APP_SECRET
  facebook:
    enabled: false
    clientId: FACEBOOK_APP_ID
    clientSecret: FACEBOOK_APP_SECRET
  github:
    enabled: false
    clientId: GITHUB_CLIENT_ID
    clientSecret: GITHUB_CLIENT_SECRET
  slack:
    enabled: false
    clientId: SLACK_CLIENT_ID
    clientSecret: SLACK_CLIENT_SECRET
  ldap:
    enabled: false
    url: 'ldap://serverhost:389'
    bindDn: cn='root'
    bindCredentials: BIND_PASSWORD
    searchBase: 'o=users,o=example.com'
    searchFilter: '(uid={{username}})'
    tlsEnabled: false
    tlsCertPath: 'C:\example\root_ca_cert.crt'
  azure:
    enabled: false
    clientId: APP_ID
    clientSecret: APP_SECRET_KEY
    resource: 00000002-0000-0000-c000-000000000000
    tenant: YOUR_TENANT.onmicrosoft.com
  oauth2:
    enabled: false
    clientId: OAUTH2_CLIENT_ID
    clientSecret: OAUTH2_CLIENT_SECRET
    authorizationURL: OAUTH2_AUTH_URL
    tokenURL: OAUTH2_TOKEN_URL
sessionSecret: 8d0a7e24ee013efb4bc52691088056dd64f797493af03ee6b4517f9b906ca069
db: 'mongodb://localhost:27017/wiki-test'
git: false
features:
  linebreaks: true
  mathjax: true
externalLogging:
  bugsnag: false
  loggly: false
  papertrail: false
  rollbar: false
  sentry: false
theme:
  primary: indigo
  alt: blue-grey
  viewSource: all
  footer: blue-grey
  code:
    dark: true
    colorize: true

```

**Comment by NGPixel** [comment]
This is a known bug. You must close and re-open the assets dialog before upload a second image.

**Comment by CatzillaOrz** [comment]
I think I fixed this issue, if anyone need this to be fixed in the stable version 1.0.x, I'll pull request

**Comment by CatzillaOrz** [comment]
I fixed this bug, I'll pull request if anybody need it
